pyccl/correlation.py

```diff
@@ -37,9 +37,11 @@ def correlation(cosmo, ell, C_ell, theta, corr_type='gg', method='fftlog'):
                                      calculate the angular correlation
                                      function (in degrees).
         corr_type (string): Type of correlation function. Choices:
-                            'GG' (galaxy-galaxy), 'GL' (galaxy-shear),
-                            'L+' (shear-shear, xi+),
-                            'L-' (shear-shear, xi-).
+                            'GG' (0x0), 'GL' (0x2),
+                            'L+' (2x2, xi+),
+                            'L-' (2x2, xi-), where the numbers refer to the
```

Comment:
```suggestion
                            'L-' (2x2, xi-), where numbers refer to the
```
Do you need an article here? [asks clueless slavic speaker]

Reply:
Wow, didn't know you could commit a suggestion!
Yeah, sounds good to me [says clueless Espanish speaker].